apps/src/blockly/addons/cdoXml.js

```diff
@@ -86,14 +91,19 @@ export default function initializeBlocklyXml(blocklyWrapper) {
       const frameSvgTop = hasFrameSvg ? 35 : 0;
```

Comment:
Not part of this PR, but there are a lot of numbers here that seem mysterious - just wondering how they're decided upon.

Reply:
Agreed.